labop/container-ontology.ttl

```diff
@@ -286,8 +274,7 @@ cont:width rdf:type owl:ObjectProperty ;
 #################################################################
 
 ###  http://www.ontology-of-units-of-measure.org/resource/om-2/hasNumericalValue
-om:hasNumericalValue rdf:type owl:DatatypeProperty ;
-                     rdfs:range xsd:decimal .
+om:hasNumericalValue rdfs:range xsd:decimal .
```

Comment:
If I am reading the diff correctly, it appears an important triple got deleted here:
```
om:hasNumericalValue rdf:type owl:DatatypeProperty
```
I think SBOLFactory will silently fail to generate the `hasNumericalValue` attribute if this is missing.

Reply:
Sorry, disregard that previous comment, I just realized I am looking at container-ontology, not labop!